packages/kbn-eslint-plugin-disable/src/helpers/regex.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type Eslint from 'eslint';
+import { ArrElement } from './types';
+
+const ESLINT_DISABLE_RE = /^eslint-disable(?:-next-line|-line)?(?<rulesBlock>.*)/;
+
+export function getRulesBlockFromEslintDisableComment(
+  comment: ArrElement<Eslint.AST.Program['comments']>
```

Comment:
FYI, you don't need a special type here, just use `[0]` at the end and it will read the item type from an array type

Reply:
Good hint, changed 👍